components/coding/profileV2/JoinedDateComponent.tsx

```diff
@@ -6,12 +6,26 @@ import { User } from "./ProfileHeaderComponent";
 export type JoinedDateComponentProps = {
   user: User;
   createdAt: Date;
+  textSize: "small" | "medium" | "large";
 };
 
 export default function JoinedDateComponent({
   user,
   createdAt,
+  textSize,
 }: JoinedDateComponentProps) {
+  let textStyle;
+  switch (textSize) {
```

Comment:
haven't seen switch case very often but this seems like a good use case

Reply:
Thanks! Kari helped me out getting started with this!!